README.md

```diff
@@ -1,31 +1,695 @@
+class that is part of the Java Collections Framework.
+
+
+(1) Edit the `ConcertRepository` class to add two instance variables:
+
+- An array named `concerts` that stores `Concert` class instances.
+- An int named `currentSize` that stores the number of `Concert` objects
+  that have been added to the array.  This value may differ from the array size,
+  which represents the maximum number of concerts that may be added to the array.
+
+
+(2) Create a constructor for the `ConcertRepository` that takes one parameter named
+`maxSize`.  The constructor should initialize the array size using the `maxSize` variable.
+Note that initially all elements in the array will be `null` since the array
+stores instances of the reference type  `Concert`.
+
+For example, the call `new ConcertRepository(3)` would create an object that looks
+like the following, with an array of size 3.
+
+![new ConcertRepository of size 3](https://curriculum-content.s3.amazonaws.com/6676/project/concertrepository_size3.png)
+
+(3) Generate a getter method for the `currentSize` instance variable named `getCurrentSize`.
+
+(4) Create a method named `add` that takes one parameter and returns a `boolean`.
+The parameter type should be `Concert`.
+
+- The `currentSize` instance variable represents the number of concerts that have
+  been added to the array. Initially this value is 0.
+- If the array is not full (i.e. `currentSize` is less than the array length),
+  add the parameter object into the array using `currentSize` as the index.  Increment
+  `currentSize` and return `true` to indicate the concert was added to the repository.
+- If the array is already full, simply return `false` to indicate the concert
+  could not be added to the repository.
+
+For example, let's step through several calls to the `add` method.
+The first three calls return `true` since there is room to add a `Concert`
+in the array.  Note the value of `currentSize` is incremented
+after inserting the object into the array.
+
+`repository.add(new Concert("Artist0", 1000))`
+
+![add one concert into the repository](https://curriculum-content.s3.amazonaws.com/6676/project/concertrepository_add1.png)
+
+`repository.add(new Concert("Artist1", 2000))`
+
+![add second concert into the repository](https://curriculum-content.s3.amazonaws.com/6676/project/concertrepository_add2.png)
+
+`repository.add(new Concert("Artist2", 3000))`
+
+![add third concert into the repository](https://curriculum-content.s3.amazonaws.com/6676/project/concertrepository_add3.png)
+
+
+Calling `add()` again when the array is full results in the method returning  `false`,
+and the `ConcertRepository` object state does not change:
+
+`repository.add(new Concert("Artist3", 4000))`
+
+![fail to add fourth concert into the repository](https://curriculum-content.s3.amazonaws.com/6676/project/concertrepository_add3.png)
+
+
+(5) Create a method named `get` that takes in int parameter and returns a `Concert` object.
```

Comment:
"That takes in int parameter"

I think there should be an "an" between "in" and "int"

Reply:
fixed